### DISCUSSION TITLE
Option to enable/disable or customize text in login/signup/password-reset forms

### DISCUSSION THREAD

**Discussion by gaurav0651** [wasp-lang/wasp#2364] [Archived]
With reference to the CustomizationOptions in 'wasp/client/auth' it would be nice to provide an option to customize the text as well which is displayed on top of the auth forms i.e. login, sign-up and password reset forms. 

<img width="419" alt="Screen Shot 2024-10-31 at 11 09 10 pm" src="https://github.com/user-attachments/assets/298f95e0-e1ba-49ea-bf6a-c1c0ffd905cc">

**Comment by Martinsos** [comment]
Certainly! Duplicate of https://github.com/wasp-lang/wasp/issues/2174 though, closing, but I linked this there.
